### ISSUE TITLE
Optional chaining and nullish coalescing operator

### ISSUE THREAD

**Issue by BigPhatNerd** [victorporof/Sublime-HTMLPrettify#733]
Optional chaining (.?) and the nullish coalescing operator (??) break upon saving javascript files when using HTML/CSS/JS Prettify in Sublime Text 3.2.
`someObj.?data.results`
upon save will become
`someObj. ? data.results`

(This is a wonderful plugin. Thank you for all the hard work!)

**Comment by alexvhicks** [comment]
Anyone have a workaround / temporary fix for this?

**Comment by victorporof** [comment]
I need to update the underlying prettifier library.

**Comment by appleinautumn** [comment]
Yes this is a big issue. I have to use lodash as a workaround so that I can keep using the plugin. Please fix this. Thank you for this great plugin.

**Comment by ismailius** [comment]
Yes. Please fix this or give us a workaround. It is breaking my 5 year old workflow.

**Comment by JoseGoncalves** [comment]
Waiting also for this...

**Comment by JoseGoncalves** [comment]
This issue was fixed in [v1.13.6 of js-beautify](https://github.com/beautify-web/js-beautify/milestone/55).
Upgrading that library and rebuilding works for me.
@victorporof Can you please publish a new version with this library upgrade?

**Comment by pags** [comment]
3 years later - is there any way to fix this issue? Making an otherwise great plugin unusable.

**Comment by JoseGoncalves** [comment]
My "fix" was to switch to another plugin: https://github.com/jonlabelle/SublimeJsPrettier

**Comment by jeffery9** [comment]
what about the progress ?
